src/test/java/uk/gov/hmcts/reform/sscs/personalisation/PersonalisationTest.java

```diff
@@ -324,6 +325,7 @@ public void appealRefWillReturnCcdCaseIdWhenCaseReferenceIsNotSet() {
                 .notificationEventType(APPEAL_RECEIVED_NOTIFICATION).build(), new SubscriptionWithType(subscriptions.getAppellantSubscription(), APPELLANT));
 
         assertEquals(CASE_ID, result.get(APPEAL_REF));
+        assertEquals(CASE_ID, result.get(CASE_REFENCE_ID));
```

Comment:
I don't see a test for `INFO_REQUEST_DETAIL` in this test. 
This is to get test coverage for this method `getLatestInfoRequestDetail`